what4/src/What4/Solver/Yices.hs

```diff
@@ -492,6 +495,8 @@ instance SMTWriter Connection where
 
   pushCommand _   = const $ safeCmd "(push)"
   popCommand _    = const $ safeCmd "(pop)"
+  push2Command _   = const $ safeCmd "(push)"
+  pop2Command _    = const $ safeCmd "(pop)"
```

Comment:
This is different than the implementation for smtlib2 (i.e., I don't see the parameter) - is this something that yices just doesn't support? It would probably be better to make it an error (even if we just aren't sure it is supported) than to implement it in a way that might be very surprising

Reply:
Yeah Yices the Yices syntax doesn't name assertion frames using numerals, although I think it mostly works the same way without numerals. You're right though, best to make it an error.